docs/guides/getting_started.md

```diff
@@ -113,6 +113,12 @@ Use flavor:
 
 ```hcl
 data "openstack_compute_flavor_v2" "myflavor" {
-  name = "Standard-2-4-50"
+  flavor_id = "b7d20f15-82f1-4ed4-a12e-e60277fe955f"
```

Comment:
this is possible, but name was more understandable

Reply:
fixed